packages/chord-detect/index.ts

```diff
@@ -19,21 +19,48 @@ const namedSet = (notes: string[]) => {
   return (chroma: number) => pcToName[chroma];
 };
 
-export function detect(source: string[]): string[] {
+export function detect(source: string[], assumePerfectFifth: boolean = false): string[] {
   const notes = source.map((n) => note(n).pc).filter((x) => x);
   if (note.length === 0) {
     return [];
   }
 
-  const found: FoundChord[] = findExactMatches(notes, 1);
+  const found: FoundChord[] = findExactMatches(notes, 1, assumePerfectFifth);
 
   return found
     .filter((chord) => chord.weight)
     .sort((a, b) => b.weight - a.weight)
     .map((chord) => chord.name);
 }
 
-function findExactMatches(notes: string[], weight: number): FoundChord[] {
+// 3m 000100000000
+// 3M 000010000000
+// 5d 000000100000
+// 5P 000000010000
+// 5A 000000001000
+// 7m 000000000010
+// 7M 000000000001
+function has3x_5P_7x(chordType: ChordType) {
```

Comment:
What about something more readable (it will be minified) like:

```suggestion
function hasAnyThirdAndPerfectFifthAndAnySeven(chordType: ChordType) {
```

I'm specially missing the "And" in the name.